app/partials/dynamic-fee.jade

```diff
@@ -0,0 +1,24 @@
+.modal-header.flex-row.flex-center
+  div(ng-hide="surge")
+    h3(ng-show="currentFee < suggestedFee" translate="LOW_FEE_WARN")
+    h3(ng-show="currentFee > suggestedFee" translate="HIGH_FEE_WARN")
+  div(ng-show="surge")
+    h3(translate="SURGE_WARN")
+  helper-button(content="DYNAMIC_FEE_HELP" linktext="LEARN_MORE" link="https://en.bitcoin.it/wiki/Transaction_fees")
+.modal-body
+  .col-sm-12(ng-hide="surge")
+    p(ng-show="currentFee < suggestedFee" translate="LOW_FEE_EXPLAIN" translate-values="{ fee: currentFee, suggested: suggestedFee }")
```

Comment:
Maybe make this `minFeeThreshold` and `maxFeeThreshold` so it doesn't immediately complain if someone rounds the number up or down?